float64[7] JointVel
